schema/README.md

```diff
@@ -0,0 +1,95 @@
+
+For such correlation to be possible the industry has formulated several protocols ([OTEL](https://github.com/open-telemetry), [ECS](https://github.com/elastic/ecs), [OpenMetrics](https://github.com/OpenObservability/OpenMetrics), [Alerts]()) for communicating these signals - the Observability schemas.
+
+---
+## Schema Aware Components
+
+The role of the Observability plugin is intended to allow maximum flexibility and not imposing a strict Index structure of the data source. Nevertheless, the modern nature of distributed application and the vast amount of telemetry producers is changing this perception.
+
+Today most of the Observability solutions (splunk, datadog, dynatrace) recommend using a consolidated schema to represent the entire variance of log/trace/metrics producers.
+
+This allows monitoring, incidents investigation and corrections process to become simpler, maintainable and reproducible.
+
+A Schema-Aware visualization component is a component which assumes the existence of specific index/indices name patterns and expects these indices to have a specific structure - a schema.
+
+As an example we can see that Trace-Analytics is schema-aware since it directly assumes the traces & serviceMap indices exist and expects them to follow a specific schema.
+
+This definition doesn’t change the existing status of visualization components which are not “Schema Aware” but it only regulates which Visual components would benefit using a schema and which will be agnostic of its content.
+
+Operation Panel for example, are not “schema aware” since they don’t assume in advanced the existence of a specific index nor do they expect the index they display to have a specific structure.
+
+Schema aware visualizations such as Applications, Metrics, Alerts and Integrations will not be able to work directly with a non-standard proprietary index unless being explicitly mapped during the query execution (this capability is planned to be supported in the future).
+
+## Data Model
+
+Simple Schema for Observability needs to allow ingestion of both (OTEL/ECS) formats and internally consolidate them to best of its capabilities for presenting a unified Observability platform.
+
+The data model is highly coupled with the visual components, for example - the Application visual component & Trace analytics are directly coupled with all the Observability schemas (Logs, Traces, Spans).
+
+## Observability index naming
+
+The Observability indices would follow the recommended immutable data stream ingestion pattern using the [data_stream concepts](https://opensearch.org/docs/latest/opensearch/data-streams/)
+
+Index pattern will follow the next naming structure `{type}`-`{dataset}`-`{namespace}`
+
+ - **type**	- indicated	the observability high level types "logs", "metrics", "traces", "alerts"
```

Comment:
In the Opensearch exporter you suggested SSO_traces-dataset-namespace - does this follow this index pattern you have in the readme? Or should it just export to traces-dataset-namespace?

Reply:
good - point, the default should be traces but we need some unique name so that we wont overlap with existing index naming for customers...